### PR TITLE
[T4A5][T09-A3]Francis Cheng

### PR DESCRIPTION
**Pull request by francischeng070**
Ready for review.

### CODE REVIEW

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java:7]: This class and its methods should have header comments. If not, developers who need to inherit from this class will not know how exactly they should override the methods and other developers who write client code for this class will not know how to use it either.

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+        private Person person;
+        private Tag tag;
+        private String tagOperation;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java:11]: string is too generic for this; think how else you can represent this information.

**Review by okkhoy — COMMENTED** [review]
you seem to have missed the next part of the exercise where you need to print the tags added/deleted on exit

**Comment by okkhoy** [comment]
@francischeng070 
Some comments added. Please ack & close the PR after reading comments.

**Comment by francischeng070** [comment]
ack

**Pull request closed without merging**